src/schema/v2/homeView/helpers/isSectionDisplayable.ts

```diff
@@ -27,5 +27,10 @@ export function isSectionDisplayable(
     })
   }
 
+  // section's display pre-check
+  if (typeof section.shouldBeDisplayed === "function") {
+    isDisplayable = isDisplayable && section?.shouldBeDisplayed(context)
```

Comment:
**question**: Is the `typeof` check necessary, given that the `shouldBeDisplayed` property is already typed as a function or undefined?

```suggestion
  if (section.shouldBeDisplayed) {
    isDisplayable = isDisplayable && section.shouldBeDisplayed(context)
```

Reply:
I went back and forth on this but settled on the current version because `if (section.shouldBeDisplayed)` looked to me too much like `shouldBeDisplayed` was  just a simple boolean flag, rather than a function to be optionally executed. Just a minor readability concern for future-us, but I don't feel _too_ strongly about it.

Reply:
might as well use lodash